vivisect/qt/memory.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, *args, **kwargs):
         self.addHotKey('ctrl+S', 'viv:make:struct:again')
         self.addHotKey('ctrl+meta+S', 'viv:make:struct:multi')
         self.addHotKey('U', 'viv:undefine')
+        self.addHotKey('ctrl+p', 'viv:preview:instr')
+        self.addHotKey('ctrl+s', 'viv:save')
```

Comment:
Would the `ctrl+s` here conflict with the `ctrl+s` that the main window hooks (ie, would one preempt the other and make it so the other is not called)?

Reply:
they're the same thing.  removed. (see note on other)